src/java_bytecode/jar_file.h

```diff
@@ -1,77 +1,32 @@
-/*******************************************************************\
-
-Module: JAR File Reading
-
-Author: Daniel Kroening, kroening@kroening.com
-
-\*******************************************************************/
-
-/// \file
-/// JAR File Reading
-
-#ifndef CPROVER_JAVA_BYTECODE_JAR_FILE_H
-#define CPROVER_JAVA_BYTECODE_JAR_FILE_H
-
-#define _LARGEFILE64_SOURCE 1
-#include "miniz/miniz.h"
+#pragma once
```

Comment:
No copyright/license information == not distributable.

Also violates "Guard headers with `#ifndef CPROVER_DIRECTORIES_FILE_H`, etc"